src/stan/mcmc/hmc/nuts_classic/adapt_dense_e_nuts_classic.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef STAN_MCMC_HMC_NUTS_ADAPT_DENSE_E_NUTS_CLASSIC_HPP
```

Comment:
I'd rather not keep it around if you think the new one's going
to be better (or at least as good).  I don't want the long term
support hassle and I don't want to give people too many knobs to
twist that aren't going to have much effect.  Your call, though.

> On Apr 8, 2016, at 12:25 PM, Michael Betancourt <notifications@github.com> wrote:
> 
> In src/stan/mcmc/hmc/nuts_classic/adapt_dense_e_nuts_classic.hpp:
> 
> > @@ -0,0 +1,55 @@
> > +#ifndef STAN_MCMC_HMC_NUTS_ADAPT_DENSE_E_NUTS_CLASSIC_HPP
> 
> Yes, just to keep the slice sampler around in case someone wants it.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>